### DISCUSSION TITLE
用@RequestParams传参报错

### DISCUSSION THREAD

**Discussion by zwac00001** [zuihou/zuihou-admin-cloud#321] [02.提问]
/
![微信图片_20231222090433](https://github.com/dromara/lamp-cloud/assets/50126287/f6b8115b-03be-445a-a8af-64d27aadbd4d)
通过swagger查看接口，发现需要接收的是Lang类型参数，但传递过来的是数组
![微信图片_20231222090433](https://github.com/dromara/lamp-cloud/assets/50126287/0a89203f-5c86-4e85-a724-1ed9f11b8fb7)

**Comment by zwac00001** [comment]
![微信图片_20231222090433](https://github.com/dromara/lamp-cloud/assets/50126287/24152662-3365-4183-b990-2a5688c6f1b3)

**Comment by zuihou** [comment]
1. 代码咋写的？
2. 参数咋传的？

3. 这个应该是你不会使用@RequestParam和没有正确传递参数导致的，跟框架没太大关系，建议百度
